gitlab/tests/test_base.py

```diff
@@ -93,7 +90,6 @@ def test_pickability(self):
         self.assertIsInstance(unpickled, FakeObject)
         self.assertTrue(hasattr(unpickled, "_module"))
         self.assertEqual(unpickled._module, original_obj_module)
-        pickled2 = pickle.dumps(unpickled)
```

Comment:
I think this is supposed to be a pickle object test.

Reply:
Ok, I will rollback it. :)